samples/app/sp-mgt/gulpfile.js

```diff
@@ -1,16 +1,45 @@
-'use strict';
+"use strict";
 
-const build = require('@microsoft/sp-build-web');
+const build = require("@microsoft/sp-build-web");
+const path = require("path");
 
-build.addSuppression(`Warning - [sass] The local CSS class 'ms-Grid' is not camelCase and will not be type-safe.`);
+build.addSuppression(
+  `Warning - [sass] The local CSS class 'ms-Grid' is not camelCase and will not be type-safe.`
+);
 
 var getTasks = build.rig.getTasks;
 build.rig.getTasks = function () {
   var result = getTasks.call(build.rig);
 
-  result.set('serve', result.get('serve-deprecated'));
+  result.set("serve", result.get("serve-deprecated"));
 
   return result;
 };
 
-build.initialize(require('gulp'));
+// add babel-loader and some transforms to handle es2021 language features which are unsupported in webpack 4 by default
```

Comment:
Wow this is wild. Can we package this setup in `mgt-spfx-utils` to make it reusable without handcrafting this code? And if not, at least make it obvious in our SPFx specific documentation

Reply:
We don't gain enough given the changes we'd be making to provide the function, which then needs to be dynamically imported thanks to esm/cjs differences